### ISSUE TITLE
Elm 0.19 compatibility

### ISSUE THREAD

**Issue by simvux** [pbogut/deoplete-elm#3]
Elm update to 0.19 broke this library on both my computers, any chance of this getting ported to the new version of elm?

**Comment by pbogut** [comment]
Well, this plugin is using [elm-oracle](https://github.com/ElmCast/elm-oracle)  to provide completion. As far as I can tell elm-oracle was not updated to handle 0.19 yet (wondering if it will be). You can find related issue [here #26](https://github.com/ElmCast/elm-oracle/issues/26).

As for this plugin I don't think there is any need for change, it should work once elm-oracle is updated.

**Comment by simvux** [comment]
I don't think it will work when elm-oracle is updated, this plugin looks for the "elm-package.json" file which has now been renamed to "elm.json". Other than that you're probably good

**Comment by pbogut** [comment]
Oh yeah, you're correct. Will change that later, wont help too much though.

**Comment by kornicameister** [comment]
@pbogut there's https://github.com/antew/elm-lsp.
It does not support code-completion but it is worth looking at. It should be added eventually.
